Basic/Pod/DeveloperGuide.pod

```diff
@@ -57,117 +57,79 @@ This is detailed in section 4.11 of L<PDL::FAQ>.
-if you are building the module from a .pd file :
-
-B<Makefile.PL:>
+To generate a stack-trace to help us debug a problem, please first of
+all do C<make realclean>, ensure you have the latest released PDL.
+Then, if it is still happening, please follow this process:
 
 =over
 
-=item *
+=item Rerun perl Makefile.PL
 
-You must remove the line 'use PDL::Core::Dev;'.
+This will update the timestamps on Makefiles, prompting a full rebuild.
 
-=item *
+=item Edit your generated top-level Makefile
 
-The line 'PDL::Core::Dev->import();' must be present
+Add C<-g> to the C<OPTIMIZE => line to generate debug symbols (this gets
+propagated to subdirectories)
 
-=item *
+=item Run: C<make>
 
-You must change the call from C<pdlpp_postamble> to a call to
-C<pdlpp_postamble_int> (with the same arguments.)
+This builds PDL into the C<blib> directory.
 
-=item *
+=item Run: C<gdb perl -ex 'run -Mblib t/tracking.t'>
 
-It seems that most modules in the PDL source use
+In gdb, run: C<bt>
 
-  VERSION_FROM => '../../Basic/PDL.pm',
+=item Copy-paste the backtrace output
 
-but not all of them in order that their version tracks
-the PDL release version.  It is possible to maintain
-separate versioning even within the PDL source tree but
-it may make things confusing.
+This should show where the segfault originated.
 
 =back
 
-Make certain that you make these changes to each 'Makefile.PL' in
-your source tree.
-
+Then please open an issue on whichever repo is relevant with your
+C<perl -V> output and PDL version, together with the stack-trace
+created above. When you do paste such output into a GitHub issue,
+please surround such blocks with ``` on their own lines. You can check
+out how it will render with the "Preview" button above the
+issue-editing box.
 
-B<Changes to the existing PDL source tree:>
-
-=over
 
-=item *
-
-Edit the 'Makefile.PL' in the directory above your module source
-to add your module directory name to
-C<DIR => [ qw/Module1 AnotherModule / ]>.
-
-=item *
-
-Add your test files (.t files) to the PDL/t directory renaming if
-required to avoid namespace conflicts.
+=head2 Random Musings
 
-=item *
+A (small) collection of random musings to note if you feel
+the need to improve or add to PDL (please do):
 
-Does your module depend on any libraries or external
-programs ?  If so, doocument the required programs with version
-numbers in PDL/DEPENDENCIES and add the PREREQ_* option to the
-main Makefile.PL if required.
+=over
 
-=item *
+=item Organizing Commits
 
-If your module requires external libraries or header files,
-add a section to perldl.conf.  The hash values with be available
-in your module's 'Makefile.PL' as $PDL::Config{WITH_MYMODULE},...
+git supports file-by-file commits so it is helpful
+to commit your changes to git a little at a time where
+each commit corresponds to a single change.  This makes
+it easy in the log to determine what was done and to
+locate any desired commit in case of issues that need
+to be resolved.
```

Comment:
This isn't quite how things are best done - it looks like you're telling people to make one commit per *file*, which isn't right - it's one per "small, meaningful change (ideally each with an update to tests)". The reference to "file-by-file commits" needs removing and simplifying.